eng/common/TestResources/README.md

```diff
@@ -30,7 +30,7 @@ Connect-AzAccount -Subscription 'YOUR SUBSCRIPTION ID'
 eng\common\TestResources\New-TestResources.ps1 -ServiceDirectory 'search'
 ```
 
-The `OutFile` switch would be set if you are running this for a .NET project on Windows. This will save test environment settings
+The `OutFile` switch should be set if you are running this for a .NET project on Windows. This will save test environment settings
```

Comment:
Changes under "eng/common" should be made in the azure-sdk-tools repo. https://github.com/Azure/azure-sdk-tools/blob/master/doc/common/common_engsys.md#workflow

Reply:
Thanks, Wes. I'll move this change over there.

Reply:
You’re right. I had forgotten that we default now.